src/pages/Dashboard/Clusters/Detail/VMbase/index.jsx

```diff
@@ -32,68 +38,195 @@ export default class VMbasedCluster extends React.Component {
     cluster: {}
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      configJson: null
+    };
+
+    this.cache = new Cache();
+  }
+
+  componentDidUpdate = async () => {
```

Comment:
I can't see otherwhere set or remove `configJson`
maybe can rename to the function `componentDidMount`